cfme/automate/dialogs/dialog_element.py

```diff
@@ -192,10 +192,8 @@ def create(self, element_data):
             view.fill(element)
             view.ele_save_button.click()
         view.save_button.click()
-        try:
-            view.flash.assert_no_error()
-        except AssertionError:
-            raise
+        view.flash.wait_displayed(timeout=5)
```

Comment:
This works fine for 5.9, but fails for 5.10 because the save button is disabled with mandatory fields not filled.